charts/gardener-extension-provider-aws/templates/deployment.yaml

```
@@ -67,8 +71,14 @@ spec:
         - --webhook-config-namespace={{ .Release.Namespace }}
         - --webhook-config-service-port={{ .Values.webhookConfig.servicePort }}
         - --webhook-config-server-port={{ .Values.webhookConfig.serverPort }}
+        {{- if .Values.gardener.runtimeCluster.enabled }}
+        - --disable-controllers=backupentry,bastion,controlplane,infrastructure,worker,healthcheck
```

```suggestion
        - --controllers=backupbucket,dnsrecord
```

Better use positive list?